apps/mail/components/ui/calendar.tsx

```
@@ -51,6 +71,58 @@ function Calendar({ className, classNames, showOutsideDays = true, ...props }: C
+                  </option>
+                ))}
+              </select>
+            </div>
+            
+            <button
+              onClick={() => setCurrentMonth(addMonths(displayMonth, 1))}
+              className={cn(
+                buttonVariants({ variant: 'outline' }),
+                'h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100'
+              )}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)